pkg/storaged/filesystem/filesystem.jsx

```diff
@@ -84,11 +84,13 @@ export function make_filesystem_card(next, backing_block, content_block, fstab_c
     const mounted = content_block && is_mounted(client, content_block);
 
     let mp_text;
-    if (mount_point && mounted)
-        mp_text = mount_point;
-    else if (mount_point && !mounted)
-        mp_text = mount_point + " " + _("(not mounted)");
-    else
+    if (mount_point) {
+        mp_text = client.strip_mount_point_prefix(mount_point);
+        if (mp_text == false)
+            return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19352-20231205-084155-675f6df4-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)